src/scripts.js

```diff
@@ -131,13 +136,54 @@ function displayRecipe(event) {
   if (card.classList.contains('recipes-container__recipe-card')) {
     hide(homeViewSection);
     hide(allRecipesSection);
+    hide(weeklyRecipesSection);
```

Comment:
I would say to leave this addition out for now. I'll figure out how to implement it tonight on my side, since I have made changes to the search functions to allow for styling.